src/View/Helper/DocBlockHelper.php

```diff
@@ -13,7 +13,10 @@
  */
 class DocBlockHelper extends Helper
 {
-    protected $_annotationSpacing = true;
+    /**
+     * @var bool Whether to add a blank line between different class annotations
+     */
+    protected $_classAnnotationSpacing = true;
```

Comment:
changing this should be avoided if we want to make this a patch release.

Reply:
I would be fine to make it a minor release, too.

Refs: https://book.cakephp.org/3.0/en/contributing/backwards-compatibility.html#working-on-cakephp

Reply:
Is there any actual feature change which warrants a minor release? I haven't been following the issue but this seems to be related to just changing the whitespacing in generated code.

Reply:
I would also not change the name here, the added doc block info suffices IMO.